docs/content/latest/releases/releases-overview.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Supported and planned releases
+headerTitle: Supported and planned releases
+linkTitle: Supported and planned releases
+description: An overview of the latest and current stable releases, and planned releases.
+section: RELEASES
+menu:
+  latest:
+    identifier: releases-overview
+    weight: 2588 
+isTocNested: true
+showAsideToc: true
+---
+
+{{< note title="Important" >}}
+
+As of version `2.2`, Yugabyte is following a new release versioning convention for YugabyteDB releases.  For details, see [Release versioning](../versioning).
+
+{{< /note >}}
+
+## Supported stable releases
+
+The following stable release series are supported for production deployments. Starting with
+version `2.2`, Yugabyte is following a new [release versioning](../versioning) convention.
+The next stable release series is expected to be available in November 2020.
+
+**Note:** Versions `2.1`, `2.0`, and `1.3` that were released before `2.2` are also considered stable
+releases and are supported for existing production deployments.
+
+| Version | Released   | Maintenance support ends | EOL     |
+| :------ | ---------- | ------------------------ | ----------------------------- |
+| [2.2](../earlier-releases/v2.2.0)     | 2020/07/15 | 2021/07/15               | 2022/01/15 |
+| [2.1](../earlier-releases/v2.1.0)     | 2020/02/25 | 2021/02/25               | 2021/08/25 |
+| [2.0](../earlier-releases/v2.0.0)     | 2019/09/17 | 2020/09/17               | 2021/03/17 |
+| [1.3](../earlier-releases/v1.3.0)     | 2019/07/15 | 2020/07/15               | 2021/01/15 |
+
+## Latest releases
+
+The latest release series specified below is a work-in-progress and introduces
+new features and changes. Use the latest releases for development and testing. 
+Starting with version `2.2`, Yugabyte is following a new [release versioning](../versioning) convention.
+
+| Version  | Released |
+| :------  | -------- |
+| [2.3](../whats-new)      | 2020/09/08 |
+
+## Planned releases
+
+Included below are tentative release dates for upcoming planned releases, subject to change.
+For updates on what's being worked on, see [Current roadmap](https://github.com/yugabyte/yugabyte-db#current-roadmap)
+in the YugabyteDB GitHub repository README page.
+
+| Version | Planned release |
+| :------ | --------------- |
+| 2.4     | 2020/11         |
+| 2.5     | 2021/01         |
```

Comment:
These should be flipped. We will put out the 2.4 release in Jan.

Reply:
Dates updated based on discussion with Chirag and comment by @rkarthik007